### PR TITLE
Adding diff_refs in MergeRequest #291

### PR DESCRIPTION
**Pull request by tomasbjerre**
Need this as input to Discussions API.

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/DiffRef.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.gitlab4j.api.models;
+
+public class DiffRef {
+    private String base_sha;
+    private String head_sha;
+    private String start_sha;
+
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/DiffRef.java]: This library utilizes camel-case for variable names, so these should be:
```
    private String baseSha;
    private String headSha;
    private String startSha;
```

As should the getters and setters for same.

File: src/main/java/org/gitlab4j/api/models/MergeRequest.java [hunk 2]
```diff
@@ -47,6 +45,7 @@
     private Integer userNotesCount;
     private String webUrl;
     private Boolean workInProgress;
+    private DiffRef diff_refs;
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequest.java]: Use camel-case:
```
 private DiffRef diffRefs
```

Same goes for the getters and setters.

**Comment by tomasbjerre** [comment]
Resolved

**Comment by tomasbjerre** [comment]
@gmessner issues resolved.

**Comment by gmessner** [comment]
@tomasbjerre 
Thank you for your contribution

**Comment by tomasbjerre** [comment]
A new release with this included would be great!

**Pull request merged**